lib/platformAccessory.js

```diff
@@ -0,0 +1,177 @@
+var uuid = require("hap-nodejs").uuid;
+var Accessory = require("hap-nodejs").Accessory;
+var Service = require("hap-nodejs").Service;
+var Characteristic = require("hap-nodejs").Characteristic;
+
+'use strict';
+
+module.exports = {
+  PlatformAccessory: PlatformAccessory
+}
+
+function PlatformAccessory(displayName, UUID, category) {
+  if (!displayName) throw new Error("Accessories must be created with a non-empty displayName.");
+  if (!UUID) throw new Error("Accessories must be created with a valid UUID.");
+  if (!uuid.isValid(UUID)) throw new Error("UUID '" + UUID + "' is not a valid UUID. Try using the provided 'generateUUID' function to create a valid UUID from any arbitrary string, like a serial number.");
+
+  this.displayName = displayName;
+  this.UUID = UUID;
+  this.category = category || Accessory.Categories.OTHER;
+  this.services = [];
+  this.reachable = false;
+  this.context = {};
```

Comment:
Yes it was empty, but I am a bit tired and was chasing inexistent other errors. I changed identification to UUID and that is pretty stable now. 
https://github.com/snowdd1/homebridge-knx/blob/plugin-2.0/index.js#L148

Code is still ugly and full of debug infos but is is running and survives a restart. Late adding of new accessories is not in there yet, I first wanted to save the status quo before going for the new features.